### PR TITLE
[NNAPI] discard align_corners

### PR DESCRIPTION
**Pull request by pinzhenx**
Fix https://github.com/intel/webml-polyfill/issues/468

### CODE REVIEW

**Comment by pinzhenx** [comment]
PR Submission Checking Summary
---
await an available Pixel device

File: services/ml/model_impl_android.cc [hunk 1]
```diff
@@ -111,6 +111,12 @@ int32_t ModelImplAndroid::AddOperation(int32_t type,
              << "outputs(" << outputs.size()
              << "): " << VectorToString(outputs.data(), outputs.size());
 
+  if (type == mojom::RESIZE_BILINEAR && inputs.size() == 4) {
```
Comment (fujunwei) [inline services/ml/model_impl_android.cc]: How about moving those to l.166 where you don't need to cast const?

Comment (pinzhenx) [reply services/ml/model_impl_android.cc]: Done. Please take a look, thanks

File: services/ml/model_impl_android.cc [hunk 2]
```diff
@@ -164,6 +164,14 @@ void ModelImplAndroid::Finish(mojom::ModelInfoPtr model_info,
   for (size_t i = 0; i < model_info->operations.size(); ++i ) {
     DLOG(INFO) << "  operation[" << i << "]";
     const mojom::OperationPtr& operation = model_info->operations[i];
+
+    if (operation->type == mojom::RESIZE_BILINEAR &&
+        operation->inputs.size() == 4) {
+      uint32_t align_corners = operation->inputs[3];
+      operation->inputs.pop_back();
```
[the remark this answers is not in the document]
Comment (pinzhenx) [reply services/ml/model_impl_android.cc]: yes

**Comment by pinzhenx** [comment]
@fujunwei Please review the updates. Thanks

File: services/ml/model_impl_android.cc [hunk 3]
```diff
@@ -164,6 +164,14 @@ void ModelImplAndroid::Finish(mojom::ModelInfoPtr model_info,
   for (size_t i = 0; i < model_info->operations.size(); ++i ) {
     DLOG(INFO) << "  operation[" << i << "]";
     const mojom::OperationPtr& operation = model_info->operations[i];
+
+    if (operation->type == mojom::RESIZE_BILINEAR &&
+        operation->inputs.size() == 4) {
+      DLOG(WARNING) << "    discard align_corners(" << operation->inputs[3]
```
Comment (fujunwei) [inline services/ml/model_impl_android.cc]: DLOG => LOG

Comment (pinzhenx) [reply services/ml/model_impl_android.cc]: Sorry for that.

**Comment by fujunwei** [comment]
Thanks, merging.

**Pull request merged**